importer/main.py

```diff
@@ -1930,15 +142,13 @@ def main(
         logging.info("Total time: " + str(total_time.total_seconds()) + " seconds")
         exit()
 
-    # set access token
-    if access_token:
-        global global_access_token
-        global_access_token = access_token
-
     final_response = ""
 
     logging.info("Starting csv import...")
-    resource_list = read_csv(csv_file)
+    csv_path = "/".join([dir_path, csv_file])
```

Comment:
This computes the wrong `csv_path` if the provided `csv_file` is an absolute file path. It assumes all csv_file should be relative and located under the importer directory which might be inconvenient.